lib/venice/client.rb

```diff
@@ -3,64 +3,49 @@
 require 'uri'
 
 module Venice
-  ITUNES_PRODUCTION_RECEIPT_VERIFICATION_ENDPOINT = 'https://buy.itunes.apple.com/verifyReceipt'
-  ITUNES_DEVELOPMENT_RECEIPT_VERIFICATION_ENDPOINT = 'https://sandbox.itunes.apple.com/verifyReceipt'
+  class Environment < Struct.new(:name, :endpoint)
```

Comment:
Should we move this class into separate file? What do you think?

Reply:
Ok for me. Done 👍